cockroachdb/templates/job.init.yaml

```diff
@@ -16,9 +16,9 @@ metadata:
   {{- with .Values.labels }}
     {{- toYaml . | nindent 4 }}
   {{- end }}
-  annotations:
```

Comment:
I don't like this approach, helm hooks annotations should always be present or someone who wants to add new annotations will override helms one

Reply:
You can add something like extraJobAnnotations and list after helm hooks

Reply:
Hi! The purpose of this was to make it possible to remove the annotations.

I've moved this discussion into a new PR: https://github.com/cockroachdb/helm-charts/pull/206

Reply:
Got it. Thanks!